.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Suggestion to prevent md errors:
```
---
name: Feature/Enhancement request
about: Suggest an idea for this project
title: As a [stakeholder], I want to [description of the enhancement].
labels: enhancement
assignees: ""
---

## Description

**Context:** [Provide adequate background information and describe the current
status quo so anyone can understand the issue without prior knowledge of the
codebase.]

**Problem Statement:** [Clearly state the problem you are experiencing or the
improvement you are suggesting.]

**Acceptance Criteria:** [Provide clear criteria for what needs to be done to
solve the problem and close the issue. This helps developers understand the
desired outcome.]

- [ ] Criterion 1
- [ ] Criterion 2
- [ ] Criterion 3

**Additional Information** (Optional): [Include any other context, diagrams, or
resources that would help in understanding the feature or enhancement.]

```